examples/src/main/java/io/scalecube/examples/gateway/HttpStubGateway.java

```diff
@@ -12,24 +12,32 @@
 
 public class HttpStubGateway implements Gateway {
 
+  private InetSocketAddress address;
```

Comment:
Put a `final` here and create two constructors, this way you would not have such blocks of code as ` .map(tick -> this)`  and assume that `this ` already contains what we want. Better build logic around immutable thing.

Reply:
it's an example